dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -109,16 +119,17 @@ public TeradataLogsConnector() {
     super("teradata-logs");
   }
 
-  private ImmutableList<TeradataJdbcSelectTask> createTimeSeriesTasks(ZonedInterval interval) {
-    return ImmutableList.of("ResUsageScpu", "ResUsageSpma").stream()
+  private ImmutableList<TeradataJdbcSelectTask> createTimeSeriesTasks(
+      ZonedInterval interval, @Nonnull ConnectorArguments arguments) {
+    return ImmutableList.of(RES_USAGE_SCPU_TABLE, RES_USAGE_SPMA_TABLE).stream()
         .map(
-            tableName ->
+            table ->
                 new TeradataJdbcSelectTask(
-                    createFilename("dbc." + tableName + "_", interval),
+                    createFilename(table.getDefaultValue() + "_", interval),
```

Comment:
Should we call the variable `defaultValue`? Creating the file-name with `defaultValue` even after we allow change seems a bit counter-intuitive

Reply:
We want the filename to have the same prefix that signifies that type of data within the file. Either the data is in the `ResUsageScpu` format, in which case the filename will have prefix `dbc.ResUsageScpu_`, or it is in `ResUsageSpma` format, in which case it will be `dbc.ResUsageSpma_`. In a sense the prefix is always the default value, but the source table can be changed.

I think the current code expresses this idea. If you think the code is not clear enough, could you, please, suggest how to change it?

Reply:
LGTM